packages/messaging-api-messenger/README.md

```diff
@@ -1249,6 +1250,119 @@ client.sendBatch([
 
 <br />
 
+<a id="broadcast-api" />
+
+### Broadcast API - [Official Docs](https://developers.facebook.com/docs/messenger-platform/send-messages/broadcast-messages)
+
+To use the broadcast API, your Messenger bot must have the following permissions:
+
+- `pages_messaging`
+- `pages_messaging_subscriptions`
+
+## `createMessageCreative(messages)`
+
+Param    | Type            | Description
+-------- | --------------- | -----------
+messages | `Array<Object>` | The messages to send.
+
+Example
+```js
+client.createMessageCreative([
+  {
+    "attachment":{
+      "type":"template",
+      "payload":{
+        "template_type":"generic",
+        "elements":[
+           {
+            "title":"Welcome to Our Marketplace!",
+            "image_url":"https://www.facebook.com/jaspers.png",
+            "subtitle":"Fresh fruits and vegetables. Yum.",
+            "buttons":[
+              {
+                "type":"web_url",
+                "url":"https://www.jaspersmarket.com",
+                "title":"View Website"
+              }              
+            ]      
+          }
+        ]
+      }       
+    }
+  }
+]).then(result => {
+  console.log(result);
+  // {
+  //   message_creative_id: 938461089,
+  // }
+})
+```
+
+The following message templates are not supported:
+
+- Airline boarding pass template
+- Airline check-in template
+- Airline itinerary template
+- Airline flight update template
+- Receipt template
+- Open graph template
+
+<br />
+
+## `sendBroadcastMessage(messageCreativeId, options)`
```

Comment:
is 2nd argument `options` or `custom_label_id`?

Reply:
options